### ISSUE TITLE
graph-node: /lib/x86_64-linux-gnu/libm.so.6: version `GLIBC_2.29' not found

### ISSUE THREAD

**Issue by rhlsthrm** [graphprotocol/graph-node#2790]
**Do you want to request a *feature* or report a *bug*?**
Bug

**What is the current behavior?**
Error when running the docker-compose, started happening yesterday.

```
graph-node: /lib/x86_64-linux-gnu/libm.so.6: version `GLIBC_2.29' not found
```

**If the current behavior is a bug, please provide the steps to reproduce and if possible a minimal demo of the problem.**
Run the docker-compose in this repo.

**What is the expected behavior?**
Runs without error.

**Comment by azf20** [comment]
Thanks @rhlsthrm 
@leoyvens could this be related to 0.24.1?

**Comment by rhlsthrm** [comment]
We were able to revert to 0.24.0 and get things working again.

**Comment by tintinthong** [comment]
> We were able to revert to 0.24.0 and get things working again.

is this true? im finding issues

**Comment by rokso** [comment]
> We were able to revert to 0.24.0 and get things working again.

@rhlsthrm I am having same issue in 0.24.0 too.

**Comment by garythung** [comment]
Also seeing this issue on 0.24.0 and 0.24.1. Running macOS 11.2.2 Big Sur

**Comment by garythung** [comment]
Reverting to 0.24.0 worked–I wasn't doing it correctly initially. I was reverting my GitHub repo which didn't functionally do anything.

Step to fix:

1. In `docker/docker-compose.yml` replace the `graph-node` image with `image: graphprotocol/graph-node:v0.24.0`

**Comment by tintinthong** [comment]
> 1. graphprotocol/graph-node:v0.24.0

tq @garythung. that works for me too!

**Comment by rokso** [comment]
>     1. graphprotocol/graph-node:v0.24.0

Thanks @garythung. I was reverting Github repo too which didn't do anything for me too. 😄

**Comment by azf20** [comment]
hey @rhlsthrm @tintinthong @rokso @garythung - this has now been fixed on the latest 0.24.1 docker tag, so you should be good to re-upgrade to 0.24.1! You may need to remove any precached 0.24.1 docker images. Thanks again for flagging and sorry for the trouble!
